### PR TITLE
Implementation for global Tate and Weierstrass models over specific toric base 3-folds

### PR DESCRIPTION
**Pull request by HereAround**
- Implements functionality for Tate models over concrete toric spaces.
- Code not complete/fully optimized yet (and the tests probably do not pass yet). To be updated later.

Related: Weierstrass models should be updated/adjusted to a similar state once complete.

(cc @apturner)

### CODE REVIEW

File: src/WeierstrassModels/constructors.jl [hunk 1]
```diff
@@ -21,64 +28,226 @@ export GlobalWeierstrassModel
...
+ambient space. This ring must be graded such that the Weierstrass polynomial
+is homogeneous and cuts out a Calabi-Yau hypersurface. Given this grading,
+one can perform a triangulation. This triangulation will typically
+take a long time to complete and yield a large number of candidate ambient
+spaces. Typically, one wishes to focus on those spaces which contain the
+toric base space in a manifest way. But even this criterion usually
+allows for many ambient spaces to be used.
+
+To circumvent this obstacle, this method operates in the opposite direction, i.e.
+"bottom-up". That is, it begins by extracting the rays and maximal cones of chose
```
Comment (apturner) [inline src/WeierstrassModels/constructors.jl:47]:
```suggestion
"bottom-up". That is, it begins by extracting the rays and maximal cones of the chosen
```

**Comment by HereAround** [comment]
Thank you @apturner for spotting all these typos and also fixing the convention on the Tate polynomial. I have just applied those changes. Let me clean up the history (by squashing all these minor changes into a single commit).

**Comment by HereAround** [comment]
The issues raised above should have all been addressed in a043cad67df9c73755c32378ca6c085b6cca82a2.

**Review by apturner — APPROVED** [review]
Thanks for the updates @HereAround!

**Pull request merged**